query-engine/query-engine-wasm/src/wasm/engine.rs

```diff
@@ -24,15 +24,6 @@ use tracing::{field, instrument::WithSubscriber, Instrument, Level, Span};
 use tracing_subscriber::filter::LevelFilter;
 use wasm_bindgen::prelude::wasm_bindgen;
 
-const CONNECTOR_REGISTRY: ConnectorRegistry<'_> = &[
-    #[cfg(feature = "postgresql")]
-    psl::builtin_connectors::POSTGRES,
-    #[cfg(feature = "mysql")]
-    psl::builtin_connectors::MYSQL,
-    #[cfg(feature = "sqlite")]
-    psl::builtin_connectors::SQLITE,
-];
-
```

Comment:
I needed to call `Connector::as_validated_connector()`, which is not `const` (as no `trait` method can be `const` in Rust). So, I had to move this top-level `const` declaration to the `QueryEngine`'s constructor.